src/simulator/perftest.py

```diff
@@ -443,15 +446,19 @@ def __init__(self, argv):
         #                     nargs=1,
         #                     help="The path where the result of the run need to be stored.")
 
+        parser.add_argument('-agr', '--skipReport',
+                            action='store_false', dest='auto_gen_report', default=True,
+                            help="When set, this flag stops the automatic generation of reports after test completion.")
+
         args = parser.parse_args(argv)
         tags = parse_tags(args.tag)
-        kill_java = args.kill_java
+        # kill_java = args.kill_java
         # run_path = args.runPath
 
         tests = load_yaml_file(args.file)
         perftest = PerfTest()
 
-        perftest.run(tests, tags)
+        perftest.run(tests, tags, args.auto_gen_report)
```

Comment:
Can you align the name of the args? Now there is 'auto_gen_report' and 'skipReport'

Reply:
https://github.com/hazelcast/hazelcast-simulator/pull/2040/commits/04c2fdc45a1c82b35a32cc70b1175d7c8e4f0e2b